src/xpk/commands/workload.py

```diff
@@ -423,6 +465,10 @@ def workload_create(args) -> None:
         gpu_rxdm_image=get_gpu_rxdm_image(system),
         gpu_rxdm_cmd=get_gpu_rxdm_cmd(system),
         gpu_tcp_volume=get_gpu_tcp_volume(system),
+        storage_volumes=get_storage_volumes_yaml_for_gpu(storages),
+        storage_volume_mounts=get_storage_volume_mounts_yaml_for_gpu(storages),
+        gcs_fuse_annotation=gcs_fuse_annotation,
+        service_account=XPK_SA,
```

Comment:
Why is it has a fixed value of XPK_SA and below we use service_account as a value?

Reply:
Fixed